src/m365/aad/commands/user/User.ts

```diff
@@ -0,0 +1,13 @@
+export interface User {
```

Comment:
As we're getting User from MS Graph, let's use the type from the @microsoft/microsoft-graph-types package instead of defining our own.

Reply:
Oops! I will correct. Thank you.